src/pybind/mgr/orchestrator/module.py

```diff
@@ -881,13 +881,25 @@ def ukn(s: Optional[str]) -> str:
 
             return HandleCommandResult(stdout=table.get_string())
 
-    @_cli_write_command('orch prometheus access info')
+    @_cli_write_command('orch prometheus set-credentials')
+    def _set_prometheus_access_info(self, user: str, password: str) -> HandleCommandResult:
```

Comment:
usually with these types of commands that allow setting a password, it's nice to provide an option to do it through an input file. You could look at how registry-login is done (it's entirely within the cephadm/module.py but the behavior could be the same here in orchestrator/module.py)  for an example.

Reply:
Done :+1:

Reply:
Added the support to read the password from a file :+1: